cypress.config.ts

```diff
@@ -34,6 +35,12 @@ export default defineConfig({
     },
 
     component: {
+        setupNodeEvents(on: Cypress.PluginEvents, config: Cypress.PluginConfigOptions) {
```

Comment:
What does this do?

Reply:
It allows you to test the content of the PDF with pdfParse.

Reply:
Cool